tests/src/LinkParser.cpp

```diff
@@ -56,6 +87,12 @@ TEST(LinkParser, parseDomainLinks)
         {"", "https.cat"},
         {"", "httpsd.cat"},
         {"", "http.cat", "/200"},
+        {"", "http.cat", "/200("},
+        {"", "chatterino.com><chatterino.com"},
+        {"https://", "chatterino.com><chatterino.com"},
```

Comment:
In markdown, this gets parsed as 2 links. We handle it as 1 link - that feels wrong

Reply:
> In markdown, this gets parsed as 2 links.

Only if the links contain a protocol (`<http://example.com><http://example.com>` → <http://example.com><http://example.com> | `<http://example.com><example.com>` → <http://example.com><example.com> | `<example.com><example.com>` → <example.com><example.com>).

I think we should reject all of them (that's much easier).

> If fixing this is out of the scope of this PR, that's fine, but I don't think they should be in the list of "valid links" if that's the case. I'd rather have them commented out in the list of invalid links

Done.